plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/shipping-address.tsx

```diff
@@ -19,27 +26,43 @@ export const ShippingAddress = (): JSX.Element => {
 		select( CHECKOUT_STORE_KEY ).prefersCollection()
 	);
 
+	const hasRates = hasShippingRate( shippingRates );
+
+	const { showCalculator } = useContext( ShippingCalculatorContext );
+
 	const formattedAddress = prefersCollection
 		? getPickupLocation( shippingRates )
 		: formatShippingAddress( shippingAddress );
 
+	const deliversToLabel = hasRates
+		? __( 'Delivers to ', 'woocommerce' )
+		: __( 'No delivery options available for ', 'woocommerce' );
+
 	const addressLabel = prefersCollection
-		? /* translators: %s location. */
-		  __( 'Collection from %s', 'woocommerce' )
-		: /* translators: %s location. */
-		  __( 'Delivers to %s', 'woocommerce' );
+		? __( 'Collection from ', 'woocommerce' )
```

Comment:
I realise this might be a pain to do given the number of strings handled here, but I think this is worse for translation because you cannot control the position of address within the string.

Could we use `createInterpolateElement` instead?

Reply:
Good point, thanks I wasn't too sure about this. Will fix.

Reply:
Resolved in [65381b7](https://github.com/woocommerce/woocommerce/pull/52658/commits/65381b7390f95cfeb9d55e4ab27fd5486c12fa28)